### ISSUE TITLE
Latest release 2.6.1 not available in CF-Community Plugin Repo

### ISSUE THREAD

**Issue by andre68723** [cloudfoundry-incubator/multiapps-cli-plugin#136]
### Description
2.6.1 is not available for installation for darwin

### Your environment
```
$ cf -v
cf Version 7.2.0+be4a5ce2b.2020-12-10
$ cf install-plugin multiapps
CF-Community nach Plug-in multiapps durchsuchen...
Plug-in multiapps 2.5.1 gefunden in: CF-Community
Plug-in multiapps 2.5.1 ist bereits installiert.
$ cf repo-plugins -r CF-Community |grep multiapps
multiapps                       2.5.1     MultiApps cli plugin performing multitarget application(MTA) operations in Cloud Foundry, such as deploying, removing, viewing etc.
```

**Comment by theghost5800** [comment]
Hi @andre68723 ,

Unfortunately latest released version of multiapps plugin is not merged https://github.com/cloudfoundry/cli-plugin-repo/pull/387 in CF-Community repo yet. This is the reason that you can find only 2.5.1 from CF-Community repo. We are looking now to contact someone who have rights to merge our PR in cli-plugin-repo. Until then you can use version 2.5.1 or you can download binary from [here](https://github.com/cloudfoundry-incubator/multiapps-cli-plugin/releases/tag/v2.6.1) and follow this [instruction](https://github.com/cloudfoundry-incubator/multiapps-cli-plugin#installation) to install latest release version on your machine.
I'll update you when latest version is available in CF-Community repository.

**Comment by theghost5800** [comment]
Hi @andre68723 latest released version 2.6.2 is already released and merged in CF Community repo, so now you can update using `cf install multiapps` command.
